### PR TITLE
VIH-9225 - Add prod/staging URLs for VH and remove Resource ID from ENVs

### PR DESCRIPTION
**Pull request by chris-pateman**
### JIRA link (if applicable) ###
VIH-9225

### Change description ###
Add prod/staging URLs for VH 
remove Resource ID from ENVs

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: apps/vh/test-api/prod.yaml [hunk 1]
```diff
@@ -9,7 +9,6 @@ spec:
     java:
       ingressHost: vh-test-api.platform.hmcts.net
       environment:
```
Comment (v1ferrarij) [inline apps/vh/test-api/prod.yaml:11]: this is missing a lot of environment variable overrides.

File: apps/vh/test-api/stg.yaml [hunk 2]
```diff
@@ -9,5 +9,4 @@ spec:
     java:
       ingressHost: vh-test-api.staging.platform.hmcts.net
       environment:
```
Comment (v1ferrarij) [inline apps/vh/test-api/stg.yaml:11]: this is missing a lot of environment variable overrides.

**Review by v1ferrarij — CHANGES_REQUESTED** [review]

File: apps/vh/test-api/prod.yaml (hunk 1 above)
Comment (chris-pateman) [reply apps/vh/test-api/prod.yaml:11]: will complete in new PR

File: apps/vh/test-api/stg.yaml (hunk 2 above)
Comment (chris-pateman) [reply apps/vh/test-api/stg.yaml:11]: will complete in new PR

**Review by v1ferrarij — APPROVED** [review]
lg

**Pull request merged**